torch_brain/utils/datamodules.py

```diff
@@ -130,18 +130,22 @@ def train_dataloader(self):
         return train_loader
 
     def val_dataloader(self):
-        val_sampler = DistributedStitchingFixedWindowBatchSampler(
+        batch_size = self.cfg.eval_batch_size or self.cfg.batch_size
+
+        val_sampler = DistributedStitchingFixedWindowSampler(
             interval_dict=self.val_dataset.get_sampling_intervals(),
             window_length=self.sequence_length,
             step=self.sequence_length / 2,
-            batch_size=self.cfg.eval_batch_size or self.cfg.batch_size,
```

Comment:
why remove eval_batch_size?

Reply:
`batch_size = self.cfg.eval_batch_size or self.cfg.batch_size` is defined above on line 133